index.html

```diff
@@ -1034,13 +1038,19 @@ <h2 id="BaseAudioContext">
               <li>In case of failure, queue a task on the <a>control thread</a>
               to execute the following, and abort these steps
                 <ol>
+                  <li>Reject all promises from <a>pendingResumePromises</a> in
+                  order, then clear <a>pendingResumePromises</a>.
+                  </li>
                   <li>Reject <em>promise</em>
```

Comment:
We don't 'queue a control message` when the context is `not allowed to start` so it's not possible that `promise` was in the `pendingResumePromises`.